pkg/lib/facade/coordinator.go

```diff
@@ -1433,6 +1434,10 @@ func (c *Coordinator) GetUsersByLoginID(loginIDKey string, loginIDValue string)
 	if errors.Is(err, api.ErrIdentityNotFound) {
 		// User identity not found is not an error in this function.
 		return "", nil
+	} else if errors.Is(err, loginid.ErrNormalize) {
+		return "", api.ErrGetUsersInvalidArgument.New("invalid Login ID value")
+	} else if errors.Is(err, loginid.ErrValidate) {
+		return "", api.ErrGetUsersInvalidArgument.New("invalid Login ID key")
```

Comment:
With my proposed change, we do not need this commit.

Reply:
removed all error handling related to `loginid.ErrValidate`